### PR TITLE
🔀🌐🇫🇷 Update fr_FR wiki translations 

### PR DESCRIPTION
**Pull request by deoxis9001**

### CODE REVIEW

File: docs/fr_FR/suggestions.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ sidebarDepth: 2
 # Suggestions :bulb:
 ::: tip
 Veuillez rédiger vos suggestions le mieux possible avant de les publier. Cela nous aide à comprendre votre suggestion et nous permet de poser moins de questions.:::
```
Comment (jxn-30) [inline docs/fr_FR/suggestions.md:9]: remove the `:::` at the end of line :wink:

File: docs/fr_FR/support.md [hunk 2]
```diff
@@ -11,8 +11,11 @@ Une petite aide, qui devrait vous aider à créer une demande d'assistance.
 
 ::: tip
 Si vous avez besoin d'aide, décrivez votre problème aussi précisément que possible. Plus vous décrirez votre problème de manière précise et détaillée, plus nous pourrons vous aider rapidement et efficacement.:::
```
Comment (jxn-30) [inline docs/fr_FR/support.md:13]: remove the `:::` at the end of line :wink:

File: docs/fr_FR/support.md [hunk 3]
```diff
@@ -11,8 +11,11 @@ Une petite aide, qui devrait vous aider à créer une demande d'assistance.
...
+:::
+
 ::: warning
```
Comment (jxn-30) [inline docs/fr_FR/support.md:16]: remove the `:::` at the end of line :wink:

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: docs/fr_FR/support.md (hunk 3 above)
Comment (deoxis9001) [reply docs/fr_FR/support.md:16]: ok thank

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**